sdk/core/System.ClientModel/tests/TestFramework/Mocks/MockPipelineTransport.cs

```diff
@@ -161,15 +187,15 @@ public RetriableTransportResponse(int status)
 
         public override int Status { get; }
 
-        public override string ReasonPhrase => throw new NotImplementedException();
+        public override string ReasonPhrase => string.Empty;
```

Comment:
It would be good to make this settable on the mock type if it's needed, rather than hard-coding the value.  The latter might cause confusion to someone using the mock type in the future.